### DISCUSSION TITLE
v2.2.0 Pause & other Key Bindings

### DISCUSSION THREAD

**Discussion by NJTravelerDH** [calibx/msfsdeck#180] [Q&A] [answered]
When you program a button, in this case, Pause, I have 2 questions:
1)  is is Pause or Active Pause?
2)  in either case, what key binding are you using?

I guessing you intend it to be the "Active Pause", since that seems to be most useful while flying.  The reason I am asking is that my keyboard does not have a "Pause" key, and I have re-programmed it to be ALT+ESC.  If you intend it to be "Active Pause" then I need to reset my system to the "regular" "Pause" key, so your programming will work correctly.

Does it make sense for you to somehow add the MSFS Key Binding code somewhere so we can see/understand your intentions?

**Comment by calibx** [comment]
Hi,

As possible I use direct SimConnect events instead of keybinding to prevent that problem.
For now I only use 2 keybindings in the code https://github.com/calibx/msfsdeck/blob/develop/MsfsPlugin/MsfsPlugin/msfs/SimConnectDAO.cs#L626 for ATC Open and Flashlight as there isn't other way to trigger that. 
For the Pause, if I remember well, it is just a binding in the profile : you can change it in the loupedeck ui

**Comment by NJTravelerDH** [comment]
Thank you.  You've introduced me to something new - SimConnect.  I can see I have much more to learn, but first, I'll continue to figure out how I can make better use of my Loupedeck Live device.
So, Pause, is Pause, not Active Pause.  If you have the time and energy, perhaps you'll add an Active Pause command in the next version. 
I think I understand how you interface the code with SimConnect. Thank you again.

**Comment by calibx** [comment, marked as answer]
Hi,
I was wrong, I used 2 events for pausing the game : PAUSE_ON and PAUSE_OFF : so not key binding in the profile.
I don't see a way to Active Pause in the SDK events : https://docs.flightsimulator.com/html/Programming_Tools/Event_IDs/Miscellaneous_Events.htm
